nilearn/decoding/space_net.py

```diff
@@ -580,22 +602,39 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
         this attribute is the mask image computed automatically from the
         data `X`.
 
-    `intercept_` : narray, shape (nclasses -1,)
-         Intercept (a.k.a. bias) added to the decision function.
-         It is available only when parameter intercept is set to True.
+    `memory_` : joblib memory cache
+
+    `intercept_` : narray, shape
+        (1,) for 2 class classification problems (i.e n_classes = 2)
+        (n_classes,) for n_classes > 2
+        Intercept (a.k.a. bias) added to the decision function.
+        It is available only when parameter intercept is set to True.
 
     `cv_` : list of pairs of lists
-         List of the (n_folds,) folds. For the corresponding fold,
-         each pair is composed of two lists of indices,
-         one for the train samples and one for the test samples.
+        Each pair is the list of indices for the train and test samples
+        for the corresponding fold.
 
-    `cv_scores_` : ndarray, shape (n_alphas, n_folds) or
-                   (n_l1_ratios, n_alphas, n_folds)
+    `cv_scores_` : ndarray, shape (n_folds, n_alphas) or (n_l1_ratios, n_folds, n_alphas)
```

Comment:
What is the right way to say about -infs in cv_scores_ as ?
I would like to add a note here about -inf values